sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/FullBackupOperation.java

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The FullBackupOperation model. */
+@Fluent
+public final class FullBackupOperation {
+    /*
+     * Status of the backup operation.
+     */
+    @JsonProperty(value = "status")
+    private String status;
+
+    /*
+     * The status details of backup operation.
+     */
+    @JsonProperty(value = "statusDetails")
+    private String statusDetails;
+
+    /*
+     * Error encountered, if any, during the full backup operation.
+     */
+    @JsonProperty(value = "error")
+    private Error error;
+
+    /*
+     * The start time of the backup operation in UTC
+     */
+    @JsonProperty(value = "startTime")
+    private Long startTime;
+
+    /*
+     * The end time of the backup operation in UTC
+     */
+    @JsonProperty(value = "endTime")
+    private Long endTime;
+
+    /*
+     * Identifier for the full backup operation.
+     */
+    @JsonProperty(value = "jobId")
+    private String jobId;
+
+    /*
+     * The Azure blob storage container Uri which contains the full backup
+     */
+    @JsonProperty(value = "azureStorageBlobContainerUri")
+    private String azureStorageBlobContainerUri;
+
+    /**
+     * Get the status property: Status of the backup operation.
+     *
+     * @return the status value.
+     */
+    public String getStatus() {
+        return this.status;
+    }
+
+    /**
+     * Set the status property: Status of the backup operation.
+     *
+     * @param status the status value to set.
+     * @return the FullBackupOperation object itself.
+     */
+    public FullBackupOperation setStatus(String status) {
```

Comment:
The user should not be able to set status - or anything on this class. It's purely a return model.

Reply:
This is auto-generated code. You can refer to my comments on similar classes on [this PR](https://github.com/Azure/azure-sdk-for-java/pull/12405).